src/pages/index.tsx

```diff
@@ -46,7 +46,7 @@ function AuthShowcase() {
     <div className="mb-20 flex flex-col items-center justify-center gap-4">
       <p className="text-center text-2xl text-white">
         {sessionData && <span>Logged in as {sessionData.user?.name}</span>}
-        {/* {secretMessage && <span> - {secretMessage}</span>} */}
+        {/*<meta name="google-signin-client_id" content="YOUR_CLIENT_ID.apps.googleusercontent.com" />*/}
```

Comment:
If we take this out does this still work? This seems like something that NextAuth probably already does for us.